### DISCUSSION TITLE
The measurement model statement with one continuous and one binary variable

### DISCUSSION THREAD

**Discussion by bpanny** [cdriveraus/ctsem#54] [Q&A]
Hello,

I'm curious what changes are made to the measurement model statement when a binary variable $y_{bin}(t)$ is included. I can see this partially addressed here: https://github.com/cdriveraus/ctsem/discussions/33

The measurement model I can find explicitly written is:

$y(t) = \Gamma_i + \Lambda \eta(t) + \zeta_i(t), \text{ where } \zeta(t) \sim N(\mathbf{0},\mathbf{\Theta}) \text{ and } \Gamma \sim N(\tau,\Psi)$

Except the manifest variance for the binary variable and its covariance with the continuous variable is set = 0. Is the error ("response" residual) for the binary variable obtained through the link $p(t) = 1 / (1 + e^{-y(t)})$ and $\epsilon_{binary}(t) = y_{bin}(t) - p(t)$?

Is this right? Either way, I'm curious how the measurement model should be alternatively stated to account for the joint continuous/binary error, assuming that the latent process statement is the same (I'm using a 2x2 identity matrix for Lambda). 

Or perhaps, are the parameters estimated by MLE for continuous and binary observations according to

$y_{\text{continuous}_i} (t) \sim N(\Lambda \eta_i (t), \theta)$

$y_{\text{binary}_i}(t) \sim \text{Bernoulli}\left(\frac{1}{1 + e^{-\Lambda\eta_i(t)}}\right)$

since the manifest variance and covariance involving the binary variable is set = 0? Which I'm gleaning from my model summary where manifestmeans is set to 0.

Any help is appreciated!

**Comment by cdriveraus** [comment]
Good question, one day I will hopefully write this up properly somewhere :) I think everything you write is correct, except you seem to pose them as alternatives so I'm wondering which of us is confused! 

So, some hours have passed since I started answering this, here is the half finished paper that hopefully answers the questions :) Feedback will be very helpful!
[binary.pdf](https://github.com/cdriveraus/ctsem/files/13270044/binary.pdf)

**Reply by cdriveraus** [reply]
The residual covariance matrix (MANIFESTVAR) is computed automatically, in a state dependent manner, for binary variables. This could be better represented in the latex output but that's a job for another day... you can take the zeroes to indicate that there is no additional variance on top of that implied by the logit link, although as mentioned in the paper this is possible, perhaps I should make it the default in ctsem, I'm not sure...
